package.json

```diff
@@ -142,7 +142,7 @@
           ]
         }
       ],
-      "no-warning-comments": "off"
```

Comment:
@varCepheid I noticed you have changed this in a bunch of PR's and I was wondering why? What is the intended value, and can we keep it the same across all files?

Reply:
I don't know why I've changed it in multiple branches. The purpose is to stop making it throw errors or warnings for the TODO comments in a couple files. I can change it to "warn" if you like, but "error" seems to be the wrong approach when the TODO comments seem important.